js/actions/sources.js

```diff
@@ -66,6 +66,14 @@ function loadSources() {
     [PROMISE]: Task.spawn(function*() {
       const response = yield gThreadClient.getSources();
 
+      gThreadClient.addListener("newSource", (event, packet) => {
+        store.dispatch(actions.newSource(packet.source));
```

Comment:
These listeners used to be setup in `main.js`, which had access to the store and actions. 

It seemed like too much coupling for `thread-client` to have access to the store and actions, so I moved them here. Thoughts?